packages/config/config/devices/0x019b/tf016.json

```diff
@@ -26,12 +26,28 @@
 		"2": {
 			"label": "On/Off control",
 			"maxNodes": 8
+		},
+		"3": {
+			"$if": "firmwareVersion >= 1.92",
+			"label": "Internal temperature sensor",
+			"maxNodes": 8
+		},
+		"4": {
+			"$if": "firmwareVersion >= 1.92",
+			"label": "External temperature sensor",
+			"maxNodes": 8
+		},
+		"5": {
+			"$if": "firmwareVersion >= 1.92",
+			"label": "Floor temperature sensor",
+			"maxNodes": 8
 		}
 	},
 	"paramInformation": [
 		{
 			"#": "1",
 			"label": "Operation mode",
+			"description": "Defines operation mode of the thermostat",
```

Comment:
This is an [unnecessary](https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=parameter-descriptions) description, please remove.